features/step_definitions/stepdefs.rb

```diff
@@ -1,27 +1,26 @@
-require "watir"
-require "rspec/expectations"
+require 'rspec/expectations'
 
 Given("I am on the Google Search Page") do
-  @browser ||= Watir::Browser.new
-  @browser.goto "https://www.google.com"
+  @google_search = GoogleSearch.new(@browser)
+  @netflix_home = NetflixHome.new(@browser)
+  @google_search.visit_google_search
```

Comment:
This does work and is valid, but I want you to get used to using on_page because that how we're doing it. So it would look like this:
on_page(GoogleSearch).visit_google_search